shared/common-adapters/user-proofs.desktop.js

```diff
@@ -9,7 +9,7 @@ import {metaNone, checking as proofChecking} from '../constants/tracker'
 
 import type {Proof, Props, MissingProof} from './user-proofs'
 
-export function MissingProofRow (proof: MissingProof, style: Object): React$Element {
+function MissingProofRow (proof: MissingProof, style: Object): React$Element<*> {
```

Comment:
Can you please explain (perhaps in a commit msg) what this `<*>` does and why you're switching to it?